blogs/abstract_models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from oscar.core.compat import AUTH_USER_MODEL
+
+
+class Timestamp(models.Model):
+
+    create_date = models.DateTimeField(auto_now_add=True)
+    update_date = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class AbstractCategory(Timestamp):
+
+    name = models.CharField(max_length=64)
+
+    class Meta:
+        ordering = ['name']
+
+    def __str__(self):
+        return self.name
+
+
+class AbstractPost(Timestamp):
+    post_title = models.CharField(max_length=200)
```

Comment:
remove post prefiex, It a part of model name already for all field for example
```suggestion
    title = models.CharField(max_length=200)
```